2025-03-01 11:58:07,981 INFO    MainThread:38528 [wandb_init.py:setup_run_log_directory():637] Logging user logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250301_115807-vebkyzf0\logs\debug.log
2025-03-01 11:58:07,983 INFO    MainThread:38528 [wandb_init.py:setup_run_log_directory():638] Logging internal logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250301_115807-vebkyzf0\logs\debug-internal.log
2025-03-01 11:58:07,984 INFO    MainThread:38528 [wandb_init.py:monkeypatch_ipython():589] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002B4CF9D1790>
2025-03-01 11:58:07,985 INFO    MainThread:38528 [wandb_init.py:init():756] calling init triggers
2025-03-01 11:58:07,986 INFO    MainThread:38528 [wandb_init.py:init():761] wandb.init called with sweep_config: {}
config: {'epochs': 250, 'activation': 'leaky_relu', 'nb_layers': 4, 'nb_neurons': 96, 'dropout_rate': 0.0, 'optimizer': 'adam', 'loss': 'mse', 'metric': 'mae', 'augmentation': False, 'image_resize': 128, '_wandb': {}}
2025-03-01 11:58:07,986 INFO    MainThread:38528 [wandb_init.py:init():778] found existing run on stack: vud85t6b
2025-03-01 11:58:07,986 INFO    MainThread:38528 [wandb_run.py:_finish():2110] finishing run cyrilfrebel-none/eyeball tracker/vud85t6b
2025-03-01 11:58:07,988 INFO    MainThread:38528 [wandb_run.py:_atexit_cleanup():2350] got exitcode: 0
2025-03-01 11:58:07,989 INFO    MainThread:38528 [wandb_run.py:_restore():2332] restore
2025-03-01 11:58:07,990 INFO    MainThread:38528 [wandb_run.py:_restore():2338] restore done
2025-03-01 11:58:10,479 INFO    MainThread:38528 [wandb_run.py:_footer_history_summary_info():3963] rendering history
2025-03-01 11:58:10,481 INFO    MainThread:38528 [wandb_run.py:_footer_history_summary_info():3995] rendering summary
2025-03-01 11:58:10,484 INFO    MainThread:38528 [wandb_run.py:_footer_sync_info():3924] logging synced files
2025-03-01 11:58:10,491 INFO    MainThread:38528 [wandb_init.py:init():789] starting backend
2025-03-01 11:58:10,491 INFO    MainThread:38528 [wandb_init.py:init():793] sending inform_init request
2025-03-01 11:58:10,506 INFO    MainThread:38528 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2025-03-01 11:58:10,510 INFO    MainThread:38528 [wandb_init.py:init():808] backend started and connected
2025-03-01 11:58:10,518 INFO    MainThread:38528 [wandb_run.py:_label_probe_notebook():1196] probe notebook
2025-03-01 11:58:10,604 INFO    MainThread:38528 [wandb_init.py:init():901] updated telemetry
2025-03-01 11:58:10,614 INFO    MainThread:38528 [wandb_init.py:init():936] communicating run to backend with 90.0 second timeout
2025-03-01 11:58:11,312 INFO    MainThread:38528 [wandb_init.py:init():994] starting run threads in backend
2025-03-01 11:58:11,810 INFO    MainThread:38528 [wandb_run.py:_console_start():2385] atexit reg
2025-03-01 11:58:11,813 INFO    MainThread:38528 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-03-01 11:58:11,813 INFO    MainThread:38528 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-03-01 11:58:11,814 INFO    MainThread:38528 [wandb_run.py:_redirect():2325] Redirects installed.
2025-03-01 11:58:11,816 INFO    MainThread:38528 [wandb_init.py:init():1036] run started, returning control to user process
2025-03-01 11:58:47,884 INFO    MainThread:38528 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-01 11:58:47,884 INFO    MainThread:38528 [wandb_init.py:_pause_backend():554] pausing backend
2025-03-01 11:58:53,764 INFO    MainThread:38528 [wandb_init.py:_resume_backend():559] resuming backend
2025-03-01 11:58:53,775 INFO    MainThread:38528 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-01 11:58:53,778 INFO    MainThread:38528 [wandb_init.py:_pause_backend():554] pausing backend
2025-03-01 11:58:54,324 INFO    MainThread:38528 [wandb_init.py:_resume_backend():559] resuming backend
2025-03-01 11:58:54,333 INFO    MainThread:38528 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-01 11:58:54,334 INFO    MainThread:38528 [wandb_init.py:_pause_backend():554] pausing backend
2025-03-01 11:58:55,875 INFO    MainThread:38528 [wandb_init.py:_resume_backend():559] resuming backend
2025-03-01 11:58:55,989 INFO    MainThread:38528 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-01 11:58:55,990 INFO    MainThread:38528 [wandb_init.py:_pause_backend():554] pausing backend
2025-03-01 12:00:48,754 INFO    MainThread:38528 [wandb_init.py:_resume_backend():559] resuming backend
2025-03-01 12:00:48,756 INFO    MainThread:38528 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-01 12:00:48,756 INFO    MainThread:38528 [wandb_init.py:_pause_backend():554] pausing backend
2025-03-01 12:00:55,697 INFO    MainThread:38528 [wandb_init.py:_resume_backend():559] resuming backend
2025-03-01 12:00:55,718 INFO    MainThread:38528 [wandb_init.py:setup_run_log_directory():637] Logging user logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250301_120055-gwxbeixg\logs\debug.log
2025-03-01 12:00:55,718 INFO    MainThread:38528 [wandb_init.py:setup_run_log_directory():638] Logging internal logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250301_120055-gwxbeixg\logs\debug-internal.log
2025-03-01 12:00:55,719 INFO    MainThread:38528 [wandb_init.py:init():756] calling init triggers
2025-03-01 12:00:55,719 INFO    MainThread:38528 [wandb_init.py:init():761] wandb.init called with sweep_config: {}
config: {'epochs': 250, 'activation': 'leaky_relu', 'nb_layers': 5, 'nb_neurons': 256, 'dropout_rate': 0.0, 'optimizer': 'adam', 'loss': 'mse', 'metric': 'mae', 'augmentation': False, 'image_resize': 128, '_wandb': {}}
2025-03-01 12:00:55,719 INFO    MainThread:38528 [wandb_init.py:init():778] found existing run on stack: vebkyzf0
2025-03-01 12:00:55,719 INFO    MainThread:38528 [wandb_run.py:_finish():2110] finishing run cyrilfrebel-none/eyeball tracker/vebkyzf0
2025-03-01 12:00:55,721 INFO    MainThread:38528 [jupyter.py:save_history():458] not saving jupyter history
2025-03-01 12:00:55,721 INFO    MainThread:38528 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-01 12:00:55,721 INFO    MainThread:38528 [wandb_init.py:_jupyter_teardown():571] cleaning up jupyter logic
2025-03-01 12:00:55,721 INFO    MainThread:38528 [wandb_run.py:_atexit_cleanup():2350] got exitcode: 0
2025-03-01 12:00:55,722 INFO    MainThread:38528 [wandb_run.py:_restore():2332] restore
2025-03-01 12:00:55,723 INFO    MainThread:38528 [wandb_run.py:_restore():2338] restore done
2025-03-01 12:00:58,132 INFO    MainThread:38528 [wandb_run.py:_footer_history_summary_info():3963] rendering history
2025-03-01 12:00:58,134 INFO    MainThread:38528 [wandb_run.py:_footer_history_summary_info():3995] rendering summary
2025-03-01 12:00:58,136 INFO    MainThread:38528 [wandb_run.py:_footer_sync_info():3924] logging synced files
